insights/parsers/virt_what.py

```diff
@@ -1,17 +1,55 @@
+'''
+VirtWhat - Command ``virt-what``
+================================
+
+Parses the output of the ``virt-what`` command to check if the host is running
+in a virtual machine.
+
+Sample input::
+    kvm
+
+Examples:
+    >>> vw = shared[VirtWhat]
+    >>> vw.is_virtual
+    True
+    >>> vw.generic
+    'kvm'
+    >>> vw.has_specific
+    False
+
+Note:
+    Currently ``virt-what`` fails when running without an environment (or very
+    restricted environment) on RHEL7, and reports below error::
+
+        virt-what: virt-what-cpuid-helper program not found in $PATH
+
+'''
 from .. import Parser, parser
 
+# FIXME:
+# The logic to handle below error should be updated after the bug being fixed.
```

Comment:
@xiangce iI assume that the bug in the Insights client.  The code at line 56 looks for this in the content, are the results valid if this is present in the input?

I think it makes sense to open an issue for this to ensure that we go back and fix it once the bug has been fixed.